pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -91,14 +91,19 @@ func buildURLFromWebhookPolicy(w *autoscalingv1.WebhookPolicy) (u *url.URL, err
 		w.Service.Namespace = "default"
 	}
 
-	return createURL(scheme, w.Service.Name, w.Service.Namespace, *w.Service.Path), nil
+	return createURL(scheme, w.Service.Name, w.Service.Namespace, *w.Service.Path, w.Service.Port), nil
 }
 
 // moved to a separate method to cover it with unit tests and check that URL corresponds to a proper pattern
-func createURL(scheme, name, namespace, path string) *url.URL {
+func createURL(scheme, name, namespace, path string, port *int32) *url.URL {
+	var hostPort int32 = 8000
```

Comment:
This should probably be 80, but there's no way we can change that without breaking backward compatibility for some people, is there?

Reply:
Looks like K8s defaults to 443 because certs
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#servicereference-v1-admissionregistration-k8s-io

So maybe 80 isn't a good idea :thinking: :shrug:

Reply:
It seems like no matter what would be best we need to stick with the existing number for compatibility.

Reply:
We have already documented that if not specified, the port would be 8000.
https://agones.dev/site/docs/reference/fleetautoscaler/
```
service is a reference to the service for this webhook. Either service or url must be specified. If the webhook is running within the cluster, then you should use service. Port 8000 will be used if it is open, otherwise it is an error.
```

Reply:
Cool - just wanted to check.